### PR TITLE
Cna/schnedier electric/20220329

### PR DESCRIPTION
**Pull request by Pazesm**

### CODE REVIEW

File: 2022/0xxx/CVE-2022-0715.json [hunk 1]
```diff
@@ -90,9 +90,14 @@
     "references": {
```
Comment (cve-team) [inline 2022/0xxx/CVE-2022-0715.json:90]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/0xxx/CVE-2022-0715.json:90]: ('Expecting property name enclosed in double quotes: line 93 column 1 (char 4679)',)

File: 2022/22xxx/CVE-2022-22805.json [hunk 2]
```diff
@@ -68,9 +68,14 @@
     "references": {
```
Comment (cve-team) [inline 2022/22xxx/CVE-2022-22805.json:68]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/22xxx/CVE-2022-22805.json:68]: ('Expecting property name enclosed in double quotes: line 71 column 1 (char 2881)',)

File: 2022/22xxx/CVE-2022-22806.json [hunk 3]
```diff
@@ -68,9 +68,14 @@
     "references": {
```
Comment (cve-team) [inline 2022/22xxx/CVE-2022-22806.json:68]: An exception of type JSONDecodeError occurred. Arguments:

Comment (cve-team) [inline 2022/22xxx/CVE-2022-22806.json:68]: ('Expecting property name enclosed in double quotes: line 71 column 1 (char 2820)',)

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by anthnysingleton** [comment]
Hello @Pazesm,

Please resolve the validation errors so the request can be processed.

Thanks,
CVE Content Team

**Comment by Pazesm** [comment]
Closing out as duplicate : https://github.com/CVEProject/cvelist/pull/5270

**Pull request closed without merging**